assets-and-images.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: page
+title: Assets and Images
+permalink: /assets-and-images/
+---
+
+Flutter applications include both code and assets (sometimes called resources).
+Assets can include static data, configuration files, or anything else an
+application needs to function; in particular, assets include icons and other
+images displayed in the UI.
+
+## Specifying assets
+
+The flutter build process uses the `flutter.yaml` file to identify assets
+required by an application, as shown in the [Tutorial](/tutorial/). There are
+two sections that specify assets: the general `assets` section and the
```

Comment:
"two sections in the flutter.yaml file" to avoid ambiguity... sections in the tutorial or section in the YAML file?